tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -364,6 +378,8 @@ <h2>Continue...</h2>
   <script>
     const _DEFAULT_LEFT_PANE_WIDTH = 450;
     const _MAIN_CONTENT_LEFT_PADDING = 8;
+    const _DEFAULT_EXPLORE_CONTAINER_HEIGHT = 200;
```

Comment:
This is way too short for most screen sizes. Since a typical monitor nowadays is at least 1000px in height, maybe we should set it to at least 500px. Even better would be setting it dynamically, depending on the size of the window.

Reply:
Now, we set the default graph / tensor visualization section height to about 1/2 the available height. The minimum height for that region is still 200px though in case the user really wants to allocate substantial space for the bottom.